package.json

```diff
@@ -168,7 +168,7 @@
     "react-bootstrap": "~0.32.4",
     "react-dom": "~16.6.3",
     "react-i18next": "~8.4.0",
-    "sanitize-html": "~1.19.3",
+    "sanitize-html": "^1.19.3",
```

Comment:
can stay as it was (interesting! we were already using sanitize-html before! :slightly_smiling_face:)

Reply:
I added a config file to keep `~` by default: https://github.com/Trustroots/trustroots/commit/7052711a13a664865132e47b5fe5dc4792c3a44c